clusterloader2/testing/load/config.yaml

```diff
@@ -854,16 +846,10 @@ steps:
     Method: InClusterNetworkLatency
     Params:
       action: gather
-  - Identifier: DnsLookupLatency
-    Method: DnsLookupLatency
-    Params:
-      action: gather
-  {{if $PROMETHEUS_SCRAPE_KUBE_PROXY}}
-  - Identifier: NetworkProgrammingLatency
-    Method: NetworkProgrammingLatency
+  - Identifier: SLOMeasurement
```

Comment:
Actually - to no block on addressing the kube-proxy scraping, let's start differently:
- let's temporarily remove the NetworkProgrammingLatency from the SLO measurement (leave only DNSLookupLatency for now)
- and we will proceed with this PR (and address NetworkProgramminlatency separately)